src/rtext.c

```diff
@@ -1344,40 +1344,44 @@ Rectangle GetGlyphAtlasRec(Font font, int codepoint)
 // Get text length in bytes, check for \0 character
 unsigned int TextLength(const char *text)
 {
-    unsigned int length = 0; //strlen(text)
-
-    if (text != NULL)
-    {
-        while (*text++) length++;
-    }
+    if (!text)
+        return 0;
 
+    // use strlen since it uses vector operations
+    unsigned int length = strlen(text);
```

Comment:
The use of `strlen()` looks like a really good improvement. It appears to be a considerable performance boost for `PLATFORM_DESKTOP`.
### Test:
```
#include <stdio.h>
#include <time.h>
#include "raylib.h"

int main(void) {

    {
        unsigned int testRes = 0;
        double timeRes = 0.0f;
        clock_t timeStart = clock();
        testRes = TextLength("abcdefghijklmnopqrstuvxwyz");
        clock_t timeEnd = clock();
        timeRes = (double)(timeEnd - timeStart) / CLOCKS_PER_SEC;
        printf("textLength:  %f sec (%i)\n", timeRes, testRes);
    }

    {
        unsigned int testRes = 0;
        double timeRes = 0.0f;
        clock_t timeStart = clock();
        testRes = TextLength2("abcdefghijklmnopqrstuvxwyz");
        clock_t timeEnd = clock();
        timeRes = (double)(timeEnd - timeStart) / CLOCKS_PER_SEC;
        printf("textLength2: %f sec (%i)\n", timeRes, testRes);
    }

    return 0;
}
```
### Results for `PLATFORM_DESKTOP`:
Tested on Linux Mint 21.1 64-bit:
```
~/test$ ./test
textLength:  0.000007 sec (26)
textLength2: 0.000001 sec (26)
~/test$ ./test
textLength:  0.000007 sec (26)
textLength2: 0.000001 sec (26)
~/test$ ./test
textLength:  0.000008 sec (26)
textLength2: 0.000002 sec (26)
```
### Results for `PLATFORM_WEB`:
Unfortunately there were no performance gains on `PLATFORM_WEB`.
Tested on Firefox 115.1.0esr 64-bit and Chromium 115.0.5790.170 64-bit both running on Linux Mint 21.1 64-bit.